app/ui/components/accounts-list/accounts-list.jsx

```diff
@@ -6,15 +6,28 @@ import { removeAccount } from '../../store/actions';
 import { floorLocalFloatAmount, floorLocalAmountToK } from '../../../helpers/balance';
 import './accounts-list.scss';
 
+import { getAccounts } from '../../services/scorum';
+
 class AccountsListUI extends Component {
   constructor() {
     super();
 
     this.removeAccount = this.removeAccount.bind(this);
   }
 
-  componentDidMount() {
-    // ...
+  async componentDidMount() {
+    const { accounts } = this.props;
+    const keys = Object.keys(accounts);
+
+    if(!keys)
+      return;
+
+    const fetchedAccounts = await getAccounts(keys);
+    keys.forEach(k=>{
+      accounts[k].vp = fetchedAccounts.find(x=>x.name == k).voting_power;
+    });
+
+    this.setState(accounts);
```

Comment:
if you use setState(accounts), then you should add state declaration to constructor and set default value

constructor() {
    super();

  this.state = {
   accounts = [],
  }

...
}

Reply:
You don't need fetch accounts and save to component store.
Because you save account to the state but you dont' use it.
And also in render method we use the same object from props (from store)

Reply:
If you want to use accounts You can use this.props.accounts.